### ISSUE TITLE
invalid escape sequence \S

### ISSUE THREAD

**Issue by irmen** [leohemsted/smartypants.py#14]
(Smartypants 2.0.1, cpython 3.8)

.eggs/smartypants-2.0.1-py3.8.egg/smartypants.py:277
  /home/irmen/Projects/Tale/.eggs/smartypants-2.0.1-py3.8.egg/smartypants.py:277: SyntaxWarning: invalid escape sequence \S
    if re.match("\S", prev_token_last_char):

**Comment by mbechard** [comment]
Any reason why this PR hasn't been merged?

**Comment by irmen** [comment]
no idea the author of said PR just closed it without merging it it seems. 🤷‍♂️

**Comment by smcv** [comment]
This appears to have been fixed in #21, but that change was not yet included in a release or on PyPI.

**Comment by unitof** [comment]
Indeed, this was fixed in release 2.0.2: https://github.com/justinmayer/smartypants.py/issues/25#issuecomment-2980447643.

I think this issue should be closed.
